index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+       <meta charset="UTF-8" />
+       <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+       <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+       <link rel="stylesheet" href="style.css" />
+       <title>My Mobile Portfolio</title>
+    </head>
+    <body>
+        <header>
+                <div class=""></div>
+                <div class=""></div>
+                
+                <nav class="content">
+                    <a href="#" class="nav-branding">Bantealem</a>
+                
+                    <ul class="nav-menu">
+                        <li class="nav-item">
+                            <a href="#" class="nav-link">Mobile Portfolio</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#" class="nav-link">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#" class="nav-link">Contacts</a>
+                        </li>
+                    </ul>
+                    <div class="content2">
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
+                </nav>
+        </header>
+        <main class="container">
+            <section class="section1">
+                <div>
+                    <h1 class="h1">
+                        I’m Bantealem <br />
+                        Glad to see <br />you!
+                    </h1>
+                    <p class="paragraph">
+                        I’m a software developer! I can help you build a product , feature
+                        or website Look through some of my work and experience! If you like
+                        what you see and have a project you need coded, don’t hestiate to
+                        contact me.
+                    </p>
+                    <p class="connect">LET'S CONNECT</p>
+                    <ul id="social-media" class="links">
+                        <li>
+                            <a href="https://twitter.com/BantealemG"><img src="assets/images/twitter.png" alt="Twitter" /></a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/bantealem-geto-a301b9213/"><img
+                                    src="assets/images/linkedin.svg" alt="Linkedin" /></a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/bantealem-geto-a301b9213/"><img src="assets/images/Medium.svg"
```

Comment:
- [ ] The background image is not loading. There is no `assets` folder in your project but you are loading the images from that folder. Please either add the folder in the project or change the path to the images in the code. 

![Screen Shot 2022-07-02 at 13 19 06](https://user-images.githubusercontent.com/12524453/176996497-4952e211-333d-4dbf-818f-35b55b5b40df.png)